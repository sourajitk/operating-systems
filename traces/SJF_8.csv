1,0,5
2,0,4
3,0,3
4,0,2
5,5,5
6,5,4
7,5,3
8,5,2
9,10,5
10,10,4
11,10,3
12,10,2
13,15,5
14,15,4
15,15,3
16,15,2
17,20,5
18,20,4
19,20,3
20,20,2
