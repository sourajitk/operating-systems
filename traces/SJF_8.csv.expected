1,5
2,34
3,12
4,7
5,55
6,38
7,17
8,9
9,60
10,42
11,22
12,14
13,65
14,46
15,27
16,19
17,70
18,50
19,30
20,24
